sematic.pth

```diff
@@ -0,0 +1,6 @@
+# This enables us to use websockets and standard HTTP requests in the same server locally,
+# which is what we want. If you try to use Gunicorn to do this, gevent will complain about
+# not monkey patching early enough, unless you have the gevent monkey patch applied VERY
+# early (like user/sitecustomize).
+# Monkey patching: https://github.com/gevent/gevent/issues/1235
+import gevent.monkey; gevent.monkey.patch_all()
```

Comment:
Is this file automatically installed in `site-packages` when the wheel is installed?

Reply:
yes